features/polyExplorer/src/components/filteredEntityList/filteredEntityList.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useContext, useState } from "react";
+
+import i18n from "../../i18n.js";
+
+import EntityList from "../entityList/entityList.jsx";
+import LinkButton from "../buttons/linkButton/linkButton.jsx";
+import { ExplorerContext } from "../../context/explorer-context.jsx";
+
+// TODO: Check if scrollbar is supposed to be on top of filter button
```

Comment:
No it's supposed to start below and at some point this used to be implemented or is still?

Reply:
This was an issue I had in a very early version, forgot to remove the TODO comment :D But it was already solved, it's below as it should be.